ticf-cri-stub/src/main/java/uk/gov/di/authentication/ticf/cri/stub/lambda/entity/TICFCRIRequest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.di.authentication.ticf.cri.stub.lambda.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.util.List;
+
+public class TICFCRIRequest {
```

Comment:
We can make this a record and clean up a lot of the class-related noise here :-)

Reply:
Ah snap @mrwilson